assets/css/v2/pre_fare/reconstructed_alert.scss

```diff
@@ -0,0 +1,285 @@
+// Is this the good kind of organized, or bad?
+// Is it useful to set screen height as a variable?
+$takeover-card-height: 1600px;
+$takeover-card-footer-height: 176px;
+
+$alert-card-height: 576px;
+$alert-card-footer-height: 80px;
+
+.alert-container {
+  /* Background colors */
+  &--blue {
+    background-color: $line-color-blue;
+  }
+
+  &--red {
+    background-color: $line-color-red;
+  }
+
+  &--orange {
+    background-color: $line-color-orange;
+  }
+
+  &--yellow {
+    background-color: $alert-yellow;
+  }
+
+  &--green {
+    background-color: $line-color-green;
+  }
+
+  // Takeover fonts
+  color: #171f26;
+  font-family: Inter;
+
+  .x-large-text {
+    font-size: 200px;
+    font-weight: 800;
+    line-height: 220px;
+  }
+  .large-text {
+    font-size: 120px;
+    line-height: 144px;
+  }
+  .medium-text {
+    font-size: 72px;
+    line-height: 86px;
+  }
+  .small-text {
+    font-size: 56px;
+    line-height: 68px;
+  }
+}
+
+.alert-container--large-flex {
+  padding: 26px 28px;
+  height: 522px;
+  box-sizing: border-box;
+
+  /* Alert contents */
+  .alert-card {
+    // Move the card so that it fits in the medium flex zone slot and only its drop-shadow extends outside.
+    transform: translateX(-20px) translateY(-10px);
+  }
+
+  .alert-card__body {
+    height: $alert-card-height - $alert-card-footer-height;
+    padding: 40px 120px 20px 44px;
+    box-sizing: border-box;
+
+    .bold { font-weight: 800; }
+    .medium-bold { font-weight: 500; }
+  }
+  .alert-card__body__content {
+    padding: 56px 0 0 8px;
+    &--extra-small {
+      font-size: 40px;
+      line-height: 52px;
+      .alert-card__body__isa-icon {
+        height: 51px;
+        padding-left: 16px;
+      }
+    }
+    &--small {
+      font-size: 48px;
+      line-height: 60px;
+      .alert-card__body__isa-icon {
+        height: 51px;
+        padding-left: 16px;
+      }
+    }
+    &--large {
+      font-size: 56px;
+      line-height: 68px;
+      .alert-card__body__isa-icon {
+        height: 51px;
+        padding-left: 16px;
+      }
+    }
+    &--extra-large {
+      font-size: 72px;
+      line-height: 86px;
+      .alert-card__body__isa-icon {
+        height: 65px;
+        padding-left: 16px;
+      }
+    }
+  }
+  .alert-card__body__route-pills {
+    display: inline-flex;
+    .route-pill {
+      display: inline-block;
+      width: fit-content;
+      height: 80px;
+      &:not(:first-child) {
+        margin-left: 16px;
+      }
+    }
+    
+    .route-pill__text {
+      position: relative;
+      width: fit-content;
+      padding: 0 40px;
+      font-size: 60px;
+      line-height: 80px;
+      text-transform: uppercase;
+      font-family: HelveticaNeue-Bold;
```

Comment:
My browser is having trouble loading this font. I changed it to `font-family: "neue-haas-grotesk-text";` and that makes it look like the design. Are you having the same issue or is it just me?

Reply:
Yep, able to reproduce in FF. Thanks!